src/catalog/tests/read-write.rs

```diff
@@ -89,85 +84,6 @@ async fn test_confirm_leadership(
     Box::new(state2).expire().await;
 }
 
-#[mz_ore::test(tokio::test)]
-#[cfg_attr(miri, ignore)] //  unsupported operation: can't call foreign function `TLS_client_method` on OS `linux`
-async fn test_persist_get_and_prune_storage_usage() {
```

Comment:
Making sure we retain and then prune storage usage updates is pretty important, is it possible to keep a test that exercises this?

Reply:
So we already have a bunch of end to end tests for storage usage, and I just added another one: https://github.com/MaterializeInc/materialize/blob/a7fac4d81e3827f497a1abc7e81672112e125b32/src/environmentd/tests/server.rs#L1107-L1588

Let me know if you think there's a scenario that's not already covered. I'll also loop in the testing team.

Reply:
Well looks like one of those tests are now failing, I'll take a look tomorrow.